hera_cal/abscal_funcs.py

```diff
@@ -944,6 +1120,7 @@ def gains2calfits(calfits_fname, abscal_gains, freq_array, time_array, pol_array
         echo("saving {}".format(calfits_fname))
         uvc.write_calfits(calfits_fname, clobber=overwrite)
 
+
```

Comment:
fill_dict_nans seems unnecessarily long. Is it worthwhile to try to accomplish it more succinctly without so many if statements?

Reply:
its fast right now, so no need from a computation perspective, and from an aesthetic perspective, I don't think its an issue